src/dsp_tools/commands/xmlupload/resource_create_client.py

```diff
@@ -188,6 +193,19 @@ def _make_value(self, value: XMLValue, value_type: str) -> dict[str, Any]:
         return res
 
 
+def _make_iiif_uri_value(iiif_uri: IIIFUriInfo, res_bnode: BNode, permissions_lookup: dict[str, Permissions]) -> Graph:
+    g = Graph()
+    iiif_bn = BNode()
+    g.add((res_bnode, KNORA_API.hasStillImageFileValue, iiif_bn))
+    g.add((iiif_bn, RDF.type, KNORA_API.StillImageExternalFileValue))
+    g.add((iiif_bn, KNORA_API.fileValueHasExternalUrl, Literal(iiif_uri.value)))
+    if iiif_uri.permissions:
+        if not (per := permissions_lookup.get(iiif_uri.permissions)):
+            raise PermissionNotExistsError(f"Could not find permissions for value: {iiif_uri.permissions}")
+        g.add((iiif_bn, KNORA_API.hasPermissions, Literal(str(per))))
```

Comment:
these three lines will probably be needed on all value types, right? And they don't depend on anything specific to the value type. If so, you could extract them to a function.